### PR TITLE
WIP - Remove 'incubat(ing, ion, or)' designation and disclaimer

### PR DESCRIPTION
**Pull request by mrutkows**

### CODE REVIEW

File: commands/project.go [hunk 1]
```diff
@@ -18,10 +18,10 @@
 package commands
 
 import (
-	"github.com/apache/incubator-openwhisk-cli/wski18n"
-	"github.com/apache/incubator-openwhisk-wskdeploy/cmd"
-	"github.com/apache/incubator-openwhisk-wskdeploy/utils"
-	wskdeploy_wski18n "github.com/apache/incubator-openwhisk-wskdeploy/wski18n"
+	"github.com/apache/openwhisk-cli/wski18n"
+	"github.com/apache/openwhisk-wskdeploy/cmd"
```
Comment (rabbah) [inline commands/project.go]: do you need to fix wskdeploy first/at the same time?

Comment (mrutkows) [reply commands/project.go]: working on it...

**Comment by mrutkows** [comment]
Build is failing accessing scala tests (coverage) in Maven?

**Comment by mrutkows** [comment]
Need time to figure out how to force ```govendor``` (package) to regenerate vendor.json with SHA1 hashes (already updated build.gradle and vendor.json manually).  No docs. on how to do this, overly complicated...  and having issues running ```govendor``` locally...

Regardless, either remove this dependency (use godeps) or document the process MUCH better including the Gradle steps.

**Comment by mrutkows** [comment]
I see Dave had similar issues/complaints last month about the same and see he somehow manually put the SHA1 in vendor.json... 
https://github.com/apache/openwhisk-cli/pull/449

**Comment by rabbah** [comment]
@sciabarracom has some patches to build the cli with go and drop gradle... not sure if it would help here.

**Comment by mrutkows** [comment]
First, I installed the source code under my GOPATHl then found that following gradle build instructions in README (i.e., ```./gradlew compile -PnativeCompile```) failed with lots of classpath errors... (had already manually updated the commit hash in build.gradle so this had no effect on vendor.json).

Here is what i "think" worked...

1. ```govendor init```
2. ```govendor list``` (sanity check)
3 ```govendor sync``` (no effect)
4 ```govendor fetch +out``` (appears to have updated vendor.json with SHA1)?

**Pull request merged**